src/components/select/simple-select/simple-select.test.js

```diff
@@ -847,6 +847,17 @@ context("Tests for Simple Select component", () => {
       selectListOptionGroup("1").should("have.text", "Group one");
     });
 
+    it("should render option list with proper maxHeight value", () => {
```

Comment:
I'm not sure if we need a Cypress test for this new prop when there's already a unit test for it. But the tests are simple enough and I'd rather write too many than too few, so no harm done.